exe/Rad.hs

```diff
@@ -50,18 +51,15 @@ builtinCommands :: [String]
 builtinCommands =
     ["version", "help"]
 
--- | Lists commands available for running.
-availableCommands
-    :: [FilePath]  -- ^ Directories to search
-    -> IO [String] -- ^ Available commands
-availableCommands dirs =
-    concat <$> forM dirs (\dir -> do
-        result <- try $ do
-            xs <- listDirectory dir
-            pure $ [drop (length radPrefix) x | x <- xs, toS radPrefix `isPrefixOf` x]
-        pure $ case result of
-            Left (_ :: IOException) -> builtinCommands
-            Right xs                -> xs ++ builtinCommands)
+-- | Returns list of Radicle sub-commands located in separate binaries.
```

Comment:
```suggestion
-- | Returns list of Radicle sub-commands located in separate directories.
```

Reply:
I'm guessing..

Reply:
I meant something different. I rewrote the docs to make it clearer.